weave/index/index_sql.py

```diff
@@ -59,22 +51,29 @@ def __init__(self, file_system, pantry_path, **kwargs):
             # even though they are not used until we connect.
             # Pylint thinks this is pointless, so pylint is getting ignored.
             # pylint: disable=pointless-statement
-            os.environ["MSSQL_HOST"]
-            os.environ["MSSQL_USERNAME"]
-            os.environ["MSSQL_PASSWORD"]
+            self._sql_connection = {
+                'host': os.environ["WEAVE_SQL_HOST"],
+                'username': os.environ["WEAVE_SQL_USERNAME"],
+                'password': os.environ["WEAVE_SQL_PASSWORD"],
+                'port': os.environ.get("WEAVE_SQL_PORT", 5432),
```

Comment:
is the port 5432 a standard or something or just a random number?

Reply:
5432 is the Postgres default port. I thought it would be a good idea to keep it because it will very often be true.